source/guides/faq.md

```diff
@@ -49,6 +49,55 @@ See <https://github.com/nix-community/home-manager>
 
 ## NixOS
 
+### How to run non-nix executables?
+
+NixOS cannot run dynamically linked executables intended for generic linux environments out of the box.
+This is because, by design, it does not have a global library path, nor does it follow the traditional linux standard for where program files are stored on the filesystem.
+
+:::{note}
+The NixOS community uses the term "FHS" for traditional linux environments, which is short for "[Filesystem Hierarchy Standard](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html)".
+:::
+
+There are a few ways to resolve this mismatch in environment expectations:
+
+- Use the version packaged in nixpkgs, if there is one.
+  You can search available packages at <https://search.nixos.org/packages>.
+
+- Write a nix expression for the program to package it in your own configuration.
+  There are multiple approaches to this:
+  - Build from source.
+    Many open-source programs are highly flexible at compile time in terms of where their files go.
+    For an introduction to this, see [](../tutorials/packaging-existing-software).
+  - Set the program's ELF metadata to include paths to libraries using [`autoPatchelfHook`](https://nixos.org/manual/nixpkgs/stable/#setup-hook-autopatchelfhook).
+    Do this if building from source isn't feasible.
+  - Wrap the program to run in an FHS-like environment using [`buildFHSEnv`](https://nixos.org/manual/nixpkgs/stable/#sec-fhs-environments).
```

Comment:
```suggestion
  - Wrap the program to run in an FHS-like environment using [`buildFHSEnv`](https://nixos.org/manual/nixpkgs/stable/#sec-fhs-environments),

```

Reply:
Why make this one end in a comma, when all the other options begin with a sentence saying what the approach is, and then separate sentences with commentary?